### PR TITLE
Fix concat immutable array

### PR DESCRIPTION
**Pull request by rnt**
On `chef-13.6.4-1.el7.x86_64` the cookbook `nrpe@2.0.2` allow to set extra allowed hosts concat the attribute `node['nrpe']['allowed_hosts']`.

At least on `chef-13.7.16-1.el7.x86_64`, the concat don't work, and we can't set extra allowed hosts con nrpe server.

Also fix all rubocop problems.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Metrics/LineLength:
```
Comment (tas50) [inline .rubocop.yml]: This file can probably go away with cookstyle. We set reasonable defaults

File: recipes/configure.rb [hunk 2]
```diff
@@ -82,6 +82,6 @@
 end
 
 service node['nrpe']['service_name'] do
-  action [:start, :enable]
+  action %i(start enable)
```
Comment (tas50) [inline recipes/configure.rb]: This is another thing we disable in Rubocop because it actually makes it harder to understand what's going on

Comment (rnt) [reply recipes/configure.rb]: Ok. ;)

File: .rubocop.yml (hunk 1 above)
Comment (rnt) [reply .rubocop.yml]: Ok ;)

**Comment by thommay** [comment]
The fixes to get cookstyle coverage are good, but we shouldn't make changes to work around bugs in the chef client - https://discourse.chef.io/t/regression-in-chef-client-13-7-16/12518 . We're still working on the proper fix for this.

**Comment by rnt** [comment]
The fix in chef will take a while.
Can we mix this in the meantime?

**Comment by rnt** [comment]
Any plan to merge and release?

:crossed_fingers:

**Comment by tas50** [comment]
13.8 is out so this is no longer necessary. I'm with @thom on this one. I don't think we should be working around a bug in chef when we have the real fix.

**Pull request closed without merging**